src/content/app/species/hooks/useSpeciesAnalytics.ts

```diff
@@ -34,7 +34,7 @@ const useSpeciesAnalytics = () => {
     const speciesNameForAnalytics = getSpeciesAnalyticsName(species);
 
     trackEvent({
-      category: 'species_page',
+      category: 'remove_species',
```

Comment:
The parent component that contains the use/do not use toggle & Remove species button is called `SpeciesTitleArea`. With that, I think it would be better to use `species_title_area` as the category for this event and `trackSpeciesUse`